src/api-reference/request/v4.endpoints.workflow-resources.markdown

```diff
@@ -0,0 +1,174 @@
+---
+title: Travel Request v4 - Workflow Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+* [Workflow Resources](#workflow-resource)
+  * [Move an existing Request in the approval workflow](#Move-existing-request-in-approval-workflow)
+
+## HTTP Status Codes
```

Comment:
Move this content into the individual requests- it goes under the HTTP Response header, as per the doc template.

Reply:
Hi @sarraloew, are your talking about the HATEOAS link and the different actions from travel & approver to add just after HTTP Request instead?

Reply:
OK done

Reply:
done